src/InputView.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+public class InputView {
+    private Scanner scanner = new Scanner(System.in);
+    InputView() {
+    }
+
+    public int[] inputVer1() {
+        int playerNum = getPlayerNum();
+        int ladderHeight = getLadderHeight();
+
+        return new int[]{playerNum, ladderHeight};
```

Comment:
```suggestion
        return new int[]{getPlayerNum(), getLadderHeight()};
```

지역변수나 멤버변수는 적게 사용할 수록 좋다고 합니다.